components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -53,6 +54,9 @@ internal class MediaSessionServiceDelegate(
     private val audioFocus: AudioFocus by lazy {
         AudioFocus(context.getSystemService(Context.AUDIO_SERVICE) as AudioManager, store)
     }
+    private val notificationId by lazy {
+        SharedIdsHelper.getIdForTag(context, AbstractMediaSessionService.NOTIFICATION_TAG)
+    }
```

Comment:
Afaik this shouldn't be needed since the `NotifictionManager` allows using tags directly?

Reply:
Ah, now I get that you want to sync the ID between `startForeground()` and `NotificationManager` and `startForeground()` does not allow setting a tag?

Reply:
And this code existed before :)

Reply:
Yes, we used to call this for every notification.  But since now we only want to use the same ID as the startForeground notification we just need to call this once in the onCreate